tests/testthat/test-cff2r.R

```diff
@@ -0,0 +1,14 @@
+context("Converting CFF to DESCRIPTION")
+
+# Finding files for testing ============================== #
+findCFF <- function(pack, filename) {
+  path <- system.file(package = pack)
+  file <- file.path(path, filename)
+  return(file)
+}
+citation <- findCFF("citation", "CFF-CITATION.cff")
```

Comment:
I'd move this line into the test itself, at the moment there's no need to have this in the global namespace. I'd rename `citation`, otherwise it's a little ambiguous if you refer to the package called `citation` or the variable also called `citation`. `findCFF` is basically the same as `system.file` with another argument order, or am I missing something?
```suggestion
citationFile <- system.file("CFF-CITATION.cff", package = "citation")
```

Reply:
You're correct, that `findCFF` function was just a clone of the `findDesc` function on `test-r2cff.R`, which I found convenient to write since I wanted to read DESCRIPTION files from several different packages. In any case, I'll commit your suggestion, and probably do something similar on `test-r2cff.R` on a separate PR.